app/src/common/shared/org/mozilla/vrbrowser/utils/ServoUtils.java

```diff
@@ -32,10 +39,11 @@ public static boolean isInstanceOfServoSession(Object obj) {
     }
 
     public static GeckoSession createServoSession(Context context) {
+        boolean isOculus = BuildConfig.FLAVOR_platform.equalsIgnoreCase("oculusvr");
```

Comment:
It's better to use `SettingsStore.getInstance(context).getLayersEnabled()`

Reply:
So it will add the border if we implement layers in other platforms, or don't add the border if we disable layers in Oculus

Reply:
Addressed.